### ISSUE TITLE
Crossfading error

### ISSUE THREAD

**Issue by enigmathion** [AzuraCast/AzuraCast#1718]
Good morning, I'm noticing small flaws when setting up crossfading on the platform. When configured as Smart Mode, the emission continuously decreases and the volume increases while Normal Mode with a value of 8 seconds is cut every 5 seconds. None of the two happens when you deactivate this function!
![Anotación 2019-07-03 145947](https://user-images.githubusercontent.com/42680070/60614448-45ee8380-9da3-11e9-8c08-a3c455fa7a11.jpg)

**Comment by enigmathion** [comment]
I add, this failure is checked using live broadcast, it does not happen when it is playlist from the server

**Comment by enigmathion** [comment]
Why will they mark it as inactive if no one answered?

**Comment by SlvrEagle23** [comment]
@enigmathion The stale bot is an automated process, not a human manually going through and marking issues as "stale". All you have to do to get it to remove the "wontfix" label is just respond that it's still an issue.

In this case, this is a known issue with Liquidsoap, and we are waiting on them to release 1.4.0 so we can update to that, which will hopefully fix a number of issues with crossfading.
